src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -13,32 +13,46 @@ const Container = styled(Box)`
   user-select: none;
 `;
 
-class RangeSelector extends Component {
-  static contextType = ThemeContext;
-
-  static defaultProps = {
-    direction: 'horizontal',
-    max: 100,
-    messages: { lower: 'Lower Bounds', upper: 'Upper Bounds' },
-    min: 0,
-    opacity: 'medium',
-    size: 'medium',
-    step: 1,
-    values: [],
-  };
-
-  state = {};
-
-  containerRef = React.createRef();
-
-  componentWillUnmount() {
-    window.removeEventListener('mousemove', this.mouseMove);
-    window.removeEventListener('mouseup', this.mouseUp);
-  }
+const RangeSelector = ({
+  direction = 'horizontal',
+  max = 100,
+  messages = { lower: 'Lower Bounds', upper: 'Upper Bounds' },
+  min = 0,
+  opacity = 'medium',
+  step = 1,
+  values = [],
+  onChange,
+  color,
+  forwardRef,
+  invert,
+  round,
+  theme: propsTheme,
+  ...rest
+}) => {
+  const context = useContext(ThemeContext);
+  const theme = context || propsTheme;
+
+  const fill = direction === 'vertical' ? true : 'vertical';
```

Comment:
Maybe it would be a good idea to extract these strings to constants.

Reply:
I agree since those strings are being used more than once.